server/routerlicious/packages/services-telemetry/src/resources.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumber } from "./lumber";
+
+export enum LogLevel {
+    Error,
+    Warning,
+    Info,
+    Verbose,
+    Debug,
+}
+
+export enum LumberType {
+    Metric,
+    Log,
+}
+
+export interface ILumberjackEngine {
+    emit(lumber: Lumber<string>): void;
+}
+
+export interface ITelemetryMetadata {
+    documentId: string | undefined;
+    tenantId: string | undefined;
+    clientId: string | undefined;
+    clientSequenceNumber: number | undefined;
+    sequenceNumber: number | undefined;
```

Comment:
@tanviraumi Do you have any opinions on whether clientSequenceNumber and sequenceNumber should be made mandatory here? I was guessing that components might be aware of documentId/tenantId/clientId more often than seq numbers, so not sure if it makes sense to put those here.

Reply:
They should not be mandatory in IMO. You may never have them as well. For example, alfred and parts of deli does not know the sequence number.

Reply:
Cool, I discussed with @TriDucNgo as per my other comment and he had a good idea on how to allow us to have custom schemas for custom occasions. I have updated the pull request with that.